recipes/lmdb/all/test_package/conanfile.py

```diff
@@ -1,18 +1,26 @@
+from conan import ConanFile
+from conan.tools.build import can_run
+from conan.tools.cmake import CMake, cmake_layout
 import os
 
-from conans import ConanFile, CMake, tools
 
+class TestPackageConan(ConanFile):
+    settings = "os", "arch", "compiler", "build_type"
+    generators = "CMakeToolchain", "CMakeDeps", "VirtualRunEnv"
+    test_type = "explicit"
 
-class LmdbTestConan(ConanFile):
-    settings = "os", "compiler", "build_type", "arch"
-    generators = "cmake"
+    def layout(self):
+        cmake_layout(self)
+
+    def requirements(self):
+        self.requires(self.tested_reference_str)
 
     def build(self):
         cmake = CMake(self)
         cmake.configure()
         cmake.build()
 
     def test(self):
```

Comment:
adding path but not running any tools? seems a little odd

Reply:
?

Reply:
https://github.com/conan-io/conan-center-index/pull/13560/files#diff-915867357bc634e76de7fbc27be2cf20a0438634dcaa5a02e711b5780e61867aR84-R86

What is in the bin path that needs to be exposed? Should we not test that it works?

If this is only libraries then the recipe is confusing me

Reply:
Ok I see what you mean. It's out of scope of this PR, it was not tested before anyway.